src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarView_Partial.cs

```diff
@@ -169,7 +169,7 @@ public CalendarView()
 			m_pTodaySelectedInnerBorderBrush = Resources[c_strTodaySelectedInnerBorderBrushStorage] as Brush;
 			m_pTodayHoverBorderBrush = Resources[c_strTodayHoverBorderBrushStorage] as Brush;
 			m_pTodayPressedBorderBrush = Resources[c_strTodayPressedBorderBrushStorage] as Brush;
-			m_pTodayBackground = Resources[c_strTodayBackgroundStorage] as Brush;
+			//m_pTodayBackground = Resources[c_strTodayBackgroundStorage] as Brush;
```

Comment:
We should still fallback on the default Brush so custom styles made for UWP would also work properly with uno